src/tests/test_day2.py

```diff
@@ -26,18 +39,299 @@ def test_deploy_day2_nodes_cloud(self, cluster, new_cluster_configuration: Clust
-            args.no_proxy = new_cluster_configuration.proxy.no_proxy
-
-        day2.execute_day2_cloud_flow(new_cluster_configuration.cluster_id, args, new_cluster_configuration.is_ipv6)
+        self.execute_day2_flow(
+            cluster.api_client,
+            new_cluster_configuration.proxy,
+            new_cluster_configuration.cluster_id,
+            new_cluster_configuration.pull_secret,
+            global_variables.ssh_public_key,
+            global_variables.num_day2_workers,
+            new_cluster_configuration.is_static_ip,
+            new_cluster_configuration.is_ipv6,
+            True,
+        )
+
+    @classmethod
+    def execute_day2_flow(
+        cls,
+        api_client: Any,
+        proxy: models.Proxy,
+        cluster_id: str,
+        pull_secret: str,
+        ssh_key: str,
+        num_day2_workers: int,
+        with_static_network_config: bool,
+        has_ipv6: bool,
+        install_cluster: bool,
+    ):
+        utils.recreate_folder(consts.IMAGE_FOLDER, force_recreate=False)
+
+        cluster = api_client.cluster_get(cluster_id=cluster_id)
+        cluster_name = cluster.name
+        openshift_version = cluster.openshift_version
+        api_vip_dnsname = "api." + cluster_name + "." + cluster.base_dns_domain
+        api_vip_ip = cluster.api_vip
+
+        openshift_cluster_id = str(uuid.uuid4())
+        day2_cluster_create_params = {
+            "openshift_version": openshift_version,
+            "api_vip_dnsname": api_vip_dnsname,
+        }
+        cluster = api_client.create_day2_cluster(
+            cluster_name + "-day2", openshift_cluster_id, **day2_cluster_create_params
+        )
+        api_client.set_pull_secret(cluster.id, pull_secret)
+        cls.set_cluster_proxy(api_client, cluster.id, proxy)
+
+        cls.config_etc_hosts(api_vip_ip, api_vip_dnsname)
+
+        tf_folder = os.path.join(utils.TerraformControllerUtil.get_folder(cluster_name), consts.Platforms.BARE_METAL)
+        cls.configure_terraform(tf_folder, num_day2_workers, api_vip_ip)
+
+        static_network_config = None
+        if with_static_network_config:
+            static_network_config = static_network.generate_day2_static_network_data_from_tf(
+                tf_folder, num_day2_workers
+            )
+
+        # Generate image
+        infra_env = api_client.create_infra_env(
+            cluster_id=cluster.id,
+            name=cluster_name + "_infra-env",
+            ssh_public_key=ssh_key,
+            static_network_config=static_network_config,
+            pull_secret=pull_secret,
+            openshift_version=openshift_version,
+        )
+        # Download image
+        iso_download_url = infra_env.download_url
+        image_path = os.path.join(consts.IMAGE_FOLDER, f"{cluster_name}-installer-image.iso")
+        log.info(f"Downloading image {iso_download_url} to {image_path}")
+        utils.download_file(iso_download_url, image_path, False)
```

Comment:
what's the meaning of this ``False`` value?

Reply:
disables ssl verify